gwtp-core/gwtp-dispatch-rest/src/main/java/com/gwtplatform/dispatch/rest/rebind/serialization/JacksonMapperProviderGenerator.java

```diff
@@ -88,19 +77,12 @@ public boolean canGenerate(ExtensionContext context) {
             getLogger().debug("Jackson Mapper Provider already generated. Returning.");
         }
 
-        List<ClassDefinition> definitions = Lists.newArrayList(getClassDefinition());
-        definitions.addAll(registeredTypes.values());
-        return definitions;
-    }
-
-    @Subscribe
-    public void onRegisterSerializableType(RegisterSerializableTypeEvent event) {
-        registeredTypes.put(event.getType(), null);
+        return Lists.newArrayList(getClassDefinition());
```

Comment:
Good catch.
It was still being used in rest-dispatch rebind classes because it used Guava's event bus. With this refactor I should be able to remove it completely.